de.tudarmstadt.ukp.wikipedia.api/src/main/java/de/tudarmstadt/ukp/wikipedia/api/sweble/PlainTextConverter.java

```diff
@@ -25,42 +25,20 @@
  * (http://www.apache.org/licenses/LICENSE-2.0)
  */
 
-import java.util.LinkedList;
-import java.util.regex.Pattern;
-
-import de.fau.cs.osr.utils.StringTools;
 import de.fau.cs.osr.ptk.common.AstVisitor;
 import de.fau.cs.osr.ptk.common.ast.AstText;
-
+import de.fau.cs.osr.utils.StringTools;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
```

Comment:
Please avoid star imports. They could lead to undesired clashes.

Reply:
again, IDE thing ^